pypeit/scripts/show_2dspec.py

```diff
@@ -171,32 +172,32 @@ def main(args):
     # TODO: JFH For some reason Ginga crashes when I try to put cuts in here.
     viewer, ch = ginga.show_image(image, chname=chname_skysub, waveimg=waveimg,
                                   bitmask=mask_in) #, cuts=(cut_min, cut_max),wcs_match=True)
-
-    if sobjs is not None:
-        show_trace(sobjs, args.det, viewer, ch)
+    if not args.clean:
+        if sobjs is not None:
```

Comment:
` if not args.clean and sobjs is not None:`

Reply:
Thanks. fixed